apps/Gruntfile.js

```diff
@@ -429,7 +429,8 @@ testsContext.keys().forEach(testsContext);
     'raceInterstitial':             './src/sites/studio/pages/raceInterstitial.js',
     'layouts/_terms_interstitial':  './src/sites/studio/pages/layouts/_terms_interstitial.js',
     'maker/setup':                  './src/sites/studio/pages/maker/setup.js',
-    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js'
+    'scriptOverview':               './src/sites/studio/pages/scriptOverview.js',
+    'home/new_teacher_homepage':         './src/sites/studio/pages/home/new_teacher_homepage.js'
```

Comment:
for filenames that aren't user-facing, and don't have obvious confusion with an existing page, I think it's okay to just name them as they will be named long-term, e.g. `home/teacher_homepage'.